TabnineSelfHosted/build.gradle

```diff
@@ -0,0 +1,99 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import java.util.stream.Collectors
+
+plugins {
+    id 'java'
+    id "org.jetbrains.kotlin.jvm" version "1.4.32"
+    id 'org.jetbrains.intellij' version '1.2.0'
+    id 'org.jlleitschuh.gradle.ktlint' version "10.0.0"
+    id 'org.jlleitschuh.gradle.ktlint-idea' version "10.0.0"
+    id "com.github.sherter.google-java-format" version "0.9"
+}
+
+repositories {
+    mavenCentral()
+}
+
+group 'com.tabnineSelfHosted'
+version project.hasProperty('externalVersion') ? project.externalVersion : '1.0.6'
+
+sourceCompatibility = 9
+targetCompatibility = 9
+
+tasks.withType(KotlinCompile).configureEach {
+    kotlinOptions {
+        jvmTarget = "9"
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply plugin: "org.jlleitschuh.gradle.ktlint-idea"
+
+dependencies {
+    implementation(project(":Common"))
```

Comment:
maybe we can clean some of these dependencies here?

I mean - it is probably included in Common, right? are we sure we also need it here?